lib/notify.rb

```diff
@@ -19,4 +19,10 @@ def self.inconsistent_feature_flags
     confused_features = Features.new.all.select { |f| f.state == 'confused' }
     Slack.post_confused_features(confused_features)
   end
+
+  def self.prs_being_deployed
+    state = State.new
+    prs = Diff.pull_requests_between(state.latest_successfull_build_to('qa').commit_sha, state.latest_successfull_build_to('staging').commit_sha)
```

Comment:
Is there a way we can pick the env & SHA that's being deployed from the Azure payload? Will this work for prod otherwise?

Reply:
I was under the assumption that the change log to staging and prod would be the same as they happen in succession in most cases? Forgot to filter this call only when env is staging

Reply:
Have made a change to include diff based on the env being deployed.
https://ukgovernmentdfe.slack.com/archives/CAHBLU6ET/p1597144759429200
https://ukgovernmentdfe.slack.com/archives/CAHBLU6ET/p1597145848430500